tools/chef/cookbooks/px_dev_extras/attributes/linux_debs.rb

```diff
@@ -24,6 +24,15 @@
 default['clang']['deb_sha256'] =
   'a7c6aa046cb3a75fae2f61e1ed43abbcce6514dd91f933bdc20f6633113994ed'
```

Comment:
just realized that the location for clang needs to be updated to point to https://github.com/pixie-io/dev-artifacts/releases/tag/clang%2F15.0-pl12

Reply:
The clang-linters deb is in the same state. I'll update that one as well.

Reply:
This is fixed in the latest revision.

I also was able to upload the libtinfo5 release, so the `tools/docker/Makefile` changes are included along with the corresponding chef install from the dev-artifacts URL.